uniffi/src/interface/mod.rs

```diff
@@ -788,3 +788,41 @@ impl APIConverter<Literal> for weedle::literal::DefaultValue<'_> {
         })
     }
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum Attribute {
+    // Add valid attributes here
```

Comment:
@weslenng I'm curious what you think about using this over in #186. The idea would be to add a `ByRef` variant to the `enum Attribute` here, and a matching case in the `impl TryFrom` below it.

In your PR #186, you would then use it when determining the `by_ref` field, something like this:

```
            by_ref: Attributes::try_from(&self.attributes)?.0.iter().iter().any(|attr| match attr {
                        Attribute::ByRef => true,
                        _ => false,
                    })
```

This would help us avoid a bit of duplicate work between #186 and #31; what do you think?